src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -0,0 +1,163 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Learn best practices for good user experiences with the new autoplay policies in Chrome.
+
+{# wf_updated_on: 2017-09-06 #}
+{# wf_published_on: 2017-09-06 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: Learn best practices for good user experiences with the new autoplay policies in Chrome. #}
+{# wf_blink_components: Blink>Media #}
+
+# Autoplay Policy Changes {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's autoplay policies are about to change in 2018 and I'm here to tell
+you why and how this is going to affect video playback with sound. Spoiler
+alert: Users are going to love it!
+
+<figure>
+  <img src="https://i.imgur.com/mctNpMj.png"
+       alt="Internet memes tagged autoplay">
+  <figcaption>
+    <b>Figure 1.</b>
+    Internet memes on <a href="https://imgur.com/mctNpMj">Imgur</a>
+  </figcaption>
+</figure>
+
+## New Behaviors {: #new-behaviors }
+
+As you may have [noticed], web browsers are moving towards stricter autoplay
+policies in order to improve the web experience for users, minimize the
+incentives to install extensions that block ads, and reduce data consumption on
+expensive and/or constrained network.
+
+With these new autoplay policies, the Chrome team aims to provide a greater
+control to users over content playing in their browser. It will also benefit
+publishers who have legitimate autoplay use cases.
+
+Chrome's autoplay policies are simple:
+
+- Muted autoplay is always allowed.
+- Autoplay with sound is allowed if any of the following conditions are met:
+    - User has interacted with the site (button click, navigation, etc.)
```

Comment:
I would say 'click, tap, etc.". Navigation isn't a gesture unless you click ;)

Reply:
Done.